src/operator/quantization/quantize_graph_pass.cc

```diff
@@ -156,29 +158,54 @@ Graph QuantizeGraph(Graph &&src) {
           new_node->inputs.emplace_back(mirror_entry);
         } else if (!NeedQuantize(e.node, excluded_nodes) &&
                    (mirror_node->op() == nullptr ||
-                    mirror_node->op()->name != "_contrib_quantize")) {
+                    mirror_node->op()->name != "_contrib_quantize") &&
```

Comment:
```suggestion
                    mirror_node->op() != nnvm::Op::Get("_contrib_quantize")
```

Reply:
Thanks for the suggestion, using `entry` instead of `mirror_node` here could be a better choice.

Reply:
Anyway, I will change all the others as you suggested.

Reply:
Using `entry` is fine with me as long as you are not only comparing the op name